README.md

```diff
@@ -27,7 +27,7 @@ or reading the [rustc guide][rustcguidebuild].
 1. Make sure you have installed the dependencies:
 
    * `g++` 5.1 or later or `clang++` 3.5 or later
-   * `python` 2.7 (but not 3.x)
+   * `python` 3
```

Comment:
I would like to just say "2.7 or 3" -- we don't want to break compatibility with python 2 yet.